app/views/transactions/destroy.js.erb

```diff
@@ -1,5 +1,11 @@
-$("#<%= dom_id(@transaction) %>").remove();
-$("#<%= dom_id(@transaction, 'edit_row') %>").remove();
+// 2 = transaction + transaction edit form
```

Comment:
@Gataniel I see, it's explanation of 2 lines... Please write more human-readable comment, like '// Need to check if there are 2 lines, then reload page'